doc/radosgw/layout.rst

```diff
@@ -38,40 +39,40 @@ You can use the following commands to introspect metadata entries: ::
     
 Some variables have been used in above commands, they are:
 
-- user: Holds user information
-- bucket: Holds a mapping between bucket name and bucket instance id
-- bucket.instance: Holds bucket instance information[2]
+- _user_: Holds user information
+- _bucket_: Holds a mapping between bucket name and bucket instance id
+- _bucket.instance_: Holds bucket instance information[2]
 
-Every metadata entry is kept on a single RADOS object. See below for implementation details.
+Each metadata entry is kept in a single RADOS object. See below for implementation details.
```

Comment:
I know it's still confusing, but "on" might be the more accurate word here

Reply:
because "in" provides less of a hint we're using omap to store these

Reply:
I've reverted, thanks for the context.